src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileItemConf.scala

```diff
@@ -64,6 +64,13 @@ class FileItemConf(args: Seq[String]) extends ScallopConf(args) {
   val size = opt[Long](
     name = "size",
     descr = "Size in bytes of the file data")
+  val file = opt[File](
+    name = "file-location", short = 'l',
+    descr = "The file to be staged (only required for copying in case of non-mendeley use case)")(shouldBeFile)
+  val isMendeley = opt[Boolean](
+    name = "is-mendeley", short = 'm',
+    descr = """Stage the dataset as a "mendeley dataset"""",
+    default = Some(false))
```

Comment:
These parameters seem to be mutually exclusive, but I don't see that in the validation. May not harm but might raise false expectations if someone does not read carefully enough and gets something accepted.